pkg/registry/core/service/storage/storage.go

```diff
@@ -182,7 +181,9 @@ func (r *GenericREST) defaultSvcOnRead(service *api.Service) error {
 		service.Spec.IPFamilies = []api.IPFamily{*r.primaryIPFamily}
 
 		// headless+selectorless
-		// headless+selectorless takes both families
+		// headless+selectorless takes both families. At this stage what kind of endpoints
```

Comment:
comment feels like words missing in the middle?

" At this stage ... we don't have any way to know ... what kind of endpoint"